instrumentation-api/src/main/java/io/opentelemetry/instrumentation/api/tracer/HttpServerTracer.java

```diff
@@ -40,6 +40,7 @@
 import java.net.URISyntaxException;
 import java.util.Iterator;
 import java.util.List;
+import org.checkerframework.checker.nullness.qual.Nullable;
```

Comment:
Does IDEA recognise this annotation?

Reply:
Yeah - IDEA (and every tool I've seen) recognizes any annotation named `Nullable` regardless of package :)